### ISSUE TITLE
Performance issue with uf_trigger_fall

### ISSUE THREAD

**Issue by youhavehowie** [Facepunch/sbox-unicycle-frenzy#39]
I'm trying to create a rising water hazard and running into some performance issues, not sure if I'm just misunderstanding something or running into a bug
https://files.facepunch.com/howie/1b2411b1/sbox-dev_ZnjWl7L4bc.mp4

Here's what the setup looks like in hammer, basically pairing the water and fall trigger to the ent_platform dummy
![image](https://user-images.githubusercontent.com/11905265/155566866-67c676a3-4b3d-415d-aea1-95712bc83e11.png)
![image](https://user-images.githubusercontent.com/11905265/155566996-ed771594-8873-4cfb-9f41-f2cbc5cd21cd.png)

If I take out the fall trigger volume performance goes back to normal

**Comment by youhavehowie** [comment]
I think I narrowed this down a little more, doesn't seem to matter if the uf_trigger_fall is moving via the ent_platform because if I just position the uf_trigger_fall over part of the level and don't have it move at all it still lags.

I started deleting parts of the level one by one that the uf_trigger_fall overlapped with until it didn't lag anymore and it seems it's a result of the uf_trigger_fall overlapping too many objects at once maybe? I thought it was instances specifically but I tried to collapse all the instances that it overlapped with and it still lagged, it only stopped lagging once I deleted all said instances which there are many of(42). Wasn't sure if maybe the instances should be prefabs instead, tried that and it still lagged, seems like the only way to get it not to lag is to reduce the number of objects it's overlapping with.

I pushed the map in its current laggy state if you want to have a look
